lib/redcloth3.rb

```diff
@@ -576,16 +576,16 @@ def block_textile_lists( text )
                     end
                 end
             end
-            lines.join( "\n" )
+            lines.join("\n")
         end
     end
 
     QUOTES_RE = /(^>+([^\n]*?)(\n|$))+/m
     QUOTES_CONTENT_RE = /^([> ]+)(.*)$/m
 
-    def block_textile_quotes( text )
+    def block_textile_quotes(text)
```

Comment:
Assignment Branch Condition size for block_textile_quotes is too high. [21.63/15]